### PR TITLE
DNS seeder #246

### PR DESCRIPTION
**Pull request by MithrilMan**
Sidechain DNS node

### CODE REVIEW

File: src/FederatedPeg.sln [hunk 1]
```diff
@@ -25,7 +25,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Sidechain Generation", "Sid
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.FederatedPeg.IntegrationTests", "Stratis.FederatedPeg.IntegrationTests\Stratis.FederatedPeg.IntegrationTests.csproj", "{A8CA48FA-A725-412D-9EC4-1BA7BA7177B2}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Stratis.FederatedSidechains.AdminDashboard", "Stratis.FederatedSidechains.AdminDashboard\Stratis.FederatedSidechains.AdminDashboard.csproj", "{2D99612E-84FE-4D46-AB3C-B6A4C08828E9}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.FederatedSidechains.AdminDashboard", "Stratis.FederatedSidechains.AdminDashboard\Stratis.FederatedSidechains.AdminDashboard.csproj", "{2D99612E-84FE-4D46-AB3C-B6A4C08828E9}"
```
Comment (zeptin) [inline src/FederatedPeg.sln]: Is this related to this PR?

File: src/Stratis.SidechainDnsD/Stratis.SidechainDnsD.csproj [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+   <PropertyGroup>
+      <OutputType>Exe</OutputType>
+      <TargetFramework>netcoreapp2.1</TargetFramework>
+      <AssemblyName>Stratis.SidechainD</AssemblyName>
```
Comment (bokobza) [inline src/Stratis.SidechainDnsD/Stratis.SidechainDnsD.csproj]: Should be `Stratis.SidechainDnsD`. Same for PackageId.

File: src/FederatedPeg.sln (hunk 1 above)
Comment (MithrilMan) [reply src/FederatedPeg.sln]: as I commented on sidechain channel

@here
adding the dns project to the solution VS changed AdminDashboard GUID for its project type
from
`Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Stratis.FederatedSidechains.AdminDashboard", "Stratis.FederatedSidechains.AdminDashboard\Stratis.FederatedSidechains.AdminDashboard.csproj", "{2D99612E-84FE-4D46-AB3C-B6A4C08828E9}"`
to
`Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.FederatedSidechains.AdminDashboard", "Stratis.FederatedSidechains.AdminDashboard\Stratis.FederatedSidechains.AdminDashboard.csproj", "{2D99612E-84FE-4D46-AB3C-B6A4C08828E9}"`

is it fine to apply this change or should i manually change it back?
basically the project type of admindashboard has been migrated from
FAE04EC0-301F-11D3-BF4B-00C04F79EFBC
to
9A19103F-16F7-4668-BE54-9A1E7A4F7556
here someone reported this "issue" and it seems its correct to change that
https://github.com/dotnet/project-system/issues/3079
